css/styles.css

```diff
@@ -0,0 +1,177 @@
+.rules-container {
+    width: 95vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.hidden {
+    display: none;
+}
+
+.rules-header {
+    display: flex;
+    justify-content: center;
+    align-self: center;
+}
+
+.rules-title {
+    margin-right: 1rem;
+}
+
+.rules-heading {
+    padding-left: 2.5rem;
+}
+
+ul {
+    font-size: 1.2rem;
+}
+
+* {
+    font-family: 'Bebas Neue', sans-serif;
+    font-weight: 800;
+}
+
+h1 {
+    text-align: center;
+}
+
+hr {
+    width: 80%;
+    border: 0.1rem solid #333333;
+}
+
+.game {
+    width: 90%;
+    margin: 2rem auto;
+    display: flex;
+    flex-direction: column;
+
+}
+
+.game span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 3rem;
+    height: 3rem;
+    background-color: #CCCCCC;
+    border: 0.15rem solid #333333;
+    border-radius: 0.5rem;
+    gap: 0.2rem;
+    font-size: 1.8rem;
+}
+
+.row0, .row1, .row2,
+.row3, .row4, .row5 {
+    display: flex;
+    justify-content: center;
+    gap: 0.5rem;
+    margin: 0.1rem;
+}
+
+@media only screen and (min-width: 700px) {
```

Comment:
You have two media queries for this screen size, combine into one

Reply:
done